src/masoniteorm/schema/platforms/MSSQLPlatform.py

```diff
@@ -0,0 +1,233 @@
+                    columns="ADD " + ", ".join(add_columns).strip(),
+                )
+            )
+
+        if table.renamed_columns:
+
+            for name, column in table.get_renamed_columns().items():
+                if column.length:
+                    length = self.create_column_length(column.column_type).format(
+                        length=column.length
+                    )
+                else:
+                    length = ""
+
+                sql.append(
+                    self.rename_column_string(table.name, name, column.name).strip()
+                )
+
+        if table.dropped_columns:
+            dropped_sql = []
+
+            for name in table.get_dropped_columns():
+                dropped_sql.append(self.drop_column_string().format(name=name).strip())
+
+            sql.append(
+                self.alter_format().format(
+                    table=self.wrap_table(table.name),
+                    columns="DROP COLUMN " + ", ".join(dropped_sql),
+                )
+            )
+
+        if table.added_foreign_keys:
+            for (
+                column,
+                foreign_key_constraint,
+            ) in table.get_added_foreign_keys().items():
+                sql.append(
+                    f"ALTER TABLE {self.wrap_table(table.name)} ADD "
+                    + self.get_foreign_key_constraint_string().format(
+                        clean_column=column,
+                        column=self.wrap_table(column),
+                        table=table.name,
+                        foreign_table=foreign_key_constraint.foreign_table,
+                        foreign_column=self.wrap_table(
+                            foreign_key_constraint.foreign_column
+                        ),
+                    )
+                )
+
+        if table.dropped_foreign_keys:
+            constraints = table.dropped_foreign_keys
+            for constraint in constraints:
+                sql.append(
+                    f"ALTER TABLE {self.wrap_table(table.name)} DROP FOREIGN KEY {constraint}"
+                )
+
+        if table.added_indexes:
+            for name, index in table.added_indexes.items():
+                sql.append(
+                    "CREATE INDEX {name} ON {table}({column})".format(
+                        name=index.name, table=table.name, column=index.column
+                    )
+                )
+
+        if table.removed_indexes:
+            constraints = table.removed_indexes
+            for constraint in constraints:
+                sql.append(
+                    f"ALTER TABLE {self.wrap_table(table.name)} DROP INDEX {constraint}"
+                )
+
+        return sql
+
+    def add_column_string(self):
+        return "{name} {data_type}{length}"
+
+    def drop_column_string(self):
+        return "{name}"
+
+    def rename_column_string(self, table, old, new):
+        return f"EXEC sp_rename 'users.post', 'comment', 'COLUMN';"
```

Comment:
Guess this first testing line can be removed
```suggestion
```

Reply:
The first line is correct. the second line needs to be removed :)